<<<<<<< HEAD:recipes-multimedia/aml-audio-hal/files/0001-remove-dtv.patch
From 7e4b7589a00276f13487e6810550c5e1978759c4 Mon Sep 17 00:00:00 2001
From: Pradeep Sriram <pradeep.sriram@amlogic.com>
Date: Sat, 7 Dec 2019 12:04:54 -0800
Subject: [PATCH 1/5] remove dtv
=======
From 7b07cf60b841b6bc255fb249311b4f7e2c48349a Mon Sep 17 00:00:00 2001
From: Pradeep Sriram <pradeep.sriram@amlogic.com>
Date: Sat, 7 Dec 2019 12:04:54 -0800
Subject: [PATCH 5/5] remove dtv
>>>>>>> 7ccca6e... rdk: remove audio-hal deps on libplayer and aml-dvb:recipes-multimedia/aml-audio-hal/files/0005-remove-dtv.patch

Change-Id: Ie2206e73294b4f7fc14544a9c42264b666b028bd
---
 CMakeLists.txt       | 5 +----
 audio_hal/audio_hw.c | 9 +++++----
 2 files changed, 6 insertions(+), 8 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
<<<<<<< HEAD:recipes-multimedia/aml-audio-hal/files/0001-remove-dtv.patch
index 34559d8..05610af 100644
=======
index fabbab2..d675809 100644
>>>>>>> 7ccca6e... rdk: remove audio-hal deps on libplayer and aml-dvb:recipes-multimedia/aml-audio-hal/files/0005-remove-dtv.patch
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -40,10 +40,8 @@ add_library(audio_hal SHARED
 	audio_hal/aml_resample_wrap.cpp
 	audio_hal/audio_android_resample_api.c
 	audio_hal/audio_data_process.c
-	audio_hal/audio_dtv_ad.c
 	audio_hal/audio_format_parse.c
 	audio_hal/audio_hw.c
-	audio_hal/audio_hw_dtv.c
 	audio_hal/audio_hw_ms12.c
 	audio_hal/audio_hw_profile.c
 	audio_hal/audio_hwsync.c
@@ -89,7 +87,6 @@ set_target_properties(audio_hal PROPERTIES
 	SOVERSION 1)
 
 target_include_directories(audio_hal PRIVATE
-	$ENV{AML_AUDIO_HAL_STAGING_DIR}/usr/include/aml_dvb/am_adp
 	./include
 	./include/utils
 	./include/amlogic_AQ_tools
@@ -102,7 +99,7 @@ target_compile_options(audio_hal PRIVATE -D_GNU_SOURCE -DTV_AUDIO_OUTPUT -Wno-de
 target_link_libraries(audio_hal
 #	audioprocessing spdifenc ms12v2 ${PREBUILD_STATIC_LIBS}
 	audioprocessing spdifenc ms12v2
-	-L$ENV{AML_AUDIO_HAL_STAGING_DIR}/usr/lib/aml_libs_nf -lcutils -lamaudioutils -ltinyalsa -lamadec -lamavutils -lam_adp -llog -lexpat -lpthread -ldl -lstdc++ -Wl,-z,defs)
+	-lcutils -lamaudioutils -ltinyalsa -llog -lexpat -lpthread -ldl -lstdc++ -Wl,-z,defs)
 
 install(TARGETS audio_hal
 	LIBRARY DESTINATION "lib")
diff --git a/audio_hal/audio_hw.c b/audio_hal/audio_hw.c
<<<<<<< HEAD:recipes-multimedia/aml-audio-hal/files/0001-remove-dtv.patch
index 1d762d7..e377889 100644
=======
index 42de4e6..3e53817 100644
>>>>>>> 7ccca6e... rdk: remove audio-hal deps on libplayer and aml-dvb:recipes-multimedia/aml-audio-hal/files/0005-remove-dtv.patch
--- a/audio_hal/audio_hw.c
+++ b/audio_hal/audio_hw.c
@@ -77,7 +77,6 @@
 #include "aml_avsync_tuning.h"
 #include "aml_ng.h"
 #include "aml_audio_timer.h"
-#include "audio_dtv_ad.h"
 
 // for invoke bluetooth rc hal
 //#include "audio_hal_thunks.h"
@@ -93,13 +92,15 @@
 #include "jb_nano.h"
 #endif
 
+#ifdef ENABLE_DTV_PATCH
 // for dtv playback
 #include "audio_hw_dtv.h"
+#include "audio_dtv_ad.h"
 //#include "../bt_voice/kehwin/audio_kw.h"
+#endif
 
-
-#define ENABLE_DTV_PATCH
-#define ENABLE_TUNER_IN
+//#define ENABLE_DTV_PATCH
+//#define ENABLE_TUNER_IN
 //#define ENABLE_NOISE_GATE
 //#define ENABLE_DRC
 //#define ENABLE_EQ
-- 
2.24.1

